person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name: 'Unknown', parent_permission: true)
+    super() # this line to calls the initialize method of the parent class
```

Comment:
- [ ] Hello, @PrinceBadru! 👋 Fantastic job on your progress so far! According to the project requirements, we need you to create a method called correct_name that will retrieve and return the value of the name attribute. 🚀 Let's make sure this method smoothly and accurately handles the task at hand. 💡

```ruby
 def correct_name
       name
 end
```